scripts/tests/provisioning/migrate_vm_to_safemode.expect

```diff
@@ -16,7 +16,7 @@ set ipk_feeds [lrange $argv 2 end]
 # default, if none is specified.
 set vm_host_ip "10.0.2.2"
 # Set the login username
-set USER root
+set USER admin
```

Comment:
So we have working niauth for dunfell now?

Reply:
[We will](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Fsrc%2Frtos%2Frtosimagesd%2Frtos_nifeed%2Frtos_nifeed_main.mak&version=GBusers%2Fxhuff%2Fown_installer_atses&line=9&lineEnd=10&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) after this PR completes: https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/148906

But this nilrt.git PR is required to make that PR build, so one of them has to go first.